numba/tests/gdb/test_break_on_symbol_version.py

```diff
@@ -0,0 +1,65 @@
+# NOTE: This test is sensitive to line numbers as it checks breakpoints
+from numba import njit
```

Comment:
@sklam as I wrote this, please can you review the test file? Thanks.

Reply:
Yes, i have reviewed it and play around with it as i cherrypicked it over.

Reply:
Great, thanks for confirming.